docs/data-integrations/sap-sql-anywhere.mdx

```diff
@@ -21,9 +21,8 @@ The required arguments to establish a connection are as follows:
 * `server` sets the current server.
 
 <Tip>
-If you installed MindsDB locally via pip, you need to install all handler dependencies manually. To do so, go to the handler's folder (mindsdb/integrations/handlers/sqlany_handler) and run this command: `pip install -r requirements.txt`.
```

Comment:
Please undo this change.

Reply:
Again, the changes to this file should be undone by using `git rm sap-sql-anywhere.mdx` and commiting/pushing to this PR.

Reply:
I'm confused did you want me to add the text back into sap-sql-anywhere? Originally you said to delete it??

Reply:
ok what is the git command to push directly to a specific PR?

Reply:
I said to undo your changes - not delete the content of this file.

You can do as follows:
`git rm sap-sql-anywhere.mdx` - to delete the file
`git add *` - to add to the branch
`git commit -m "changes description"` - to commit to the branch
`git push` - to push to this PR

Please see the GitHub docs I linked earlier.